### ISSUE TITLE
AWS Redshift cluster creation stuck in Modifying state and snapshot creation stuck in Creating state

### ISSUE THREAD

**Issue by pankajkoti** [astronomer/astronomer-providers#1062]
It has been observed 2 times so far that the Redshift cluster creation as part of our DAG runs triggered via Master DAG remains in Modifying state and the corresponding snapshot remains in Creating state as seen in the screenshots believe. Open an AWS Support Case to seek help on this abnormality in our AWS account.

Cluster creation stuck in Modifying state:
<img width="1087" alt="Screenshot 2023-05-08 at 10 12 18 AM" src="https://user-images.githubusercontent.com/10206082/236735590-47aca0d4-2422-41f8-82d0-c8a110aea40f.png">

Snapshot creation stuck in Creating state:
<img width="1676" alt="Screenshot 2023-05-08 at 10 12 42 AM" src="https://user-images.githubusercontent.com/10206082/236735626-9975a295-52e0-444d-b8e9-55ccb30b61dc.png">

**Comment by pankajkoti** [comment]
I have created an AWS support case for this with Case ID 12625037521. It was marked resolved last week. But, I have reopened it today as we're seeing the same issue again.

https://support.console.aws.amazon.com/support/home#/case/?displayId=12625037521&language=en

I had also posted a question on repost.aws but we did not get a satisfactory response there and are being asked to create a support case
https://repost.aws/questions/QUR7lO9YsYRDeZUz8QnhmDgg/amazon-redshift-cluster-snapshot-stuck-in-creating-state
